### ISSUE TITLE
Cannot read property 'cells' in htmlToGridSettings helper

### ISSUE THREAD

**Issue by swistach** [handsontable/handsontable#7311]
### Description
Our helper `htmlToGridSettings` throws an error if a passed argument contains an empty table (no rows).
That helper is using in the pasting process and might have a negative impact on the `CopyPaste` plugin.

### Steps to reproduce
1. Call `Handsontable.helper.htmlToGridSettings('<table></table>)`

Result:
```
Uncaught TypeError: Cannot read property 'cells' of null
    at Object.t.htmlToGridSettings
```

### Demo
https://jsfiddle.net/qzoh2p5d/

### Your environment
* Handsontable version: 7.1.0+
* Browser Name and version: any
* Operating System: any

**Comment by aninde** [comment]
This issue will be fixed in the upcoming v8.2.0.

https://jsfiddle.net/aninde/jc8Lgxtm/

Tested on macOS Catalina 10.15.7
- Chrome Beta 87 and stable 86
- FF 83.0b7 and stable 82
- Safari 14 and Safari Technology Prev 14.1

Windows 10
- IE 11
- Edge 86
- Edge Dev 88

**Comment by AMBudnik** [comment]
Closed as solved for v 8.2.0.
